cubics-obc-test/Components/Burnwire/Burnwire.hpp

```diff
@@ -33,20 +36,33 @@ namespace Burnwire {
 
     PRIVATE:
 
+      // Component state tracking variables
+      Fw::On state; //Tracks state of whether or not burnwire is on or off. 
+      U32 count; //Incremented over a 1Hz rate group, until a certain time has passed
+      
       // ----------------------------------------------------------------------
       // Command handler implementations
       // ----------------------------------------------------------------------
 
-      //! Implementation for TODO command handler
-      //! TODO
-      void TODO_cmdHandler(
+       //! Handler implementation for run
+      //!
+      void run_handler(
+          const NATIVE_INT_TYPE portNum, /*!< The port number*/
+          NATIVE_UINT_TYPE context /*!< 
```

Comment:
Is there an fprime-util lint or indent? The comment might have wrapped because it would make the line too wide but it looks inconsistent as is. If we have lint/indent then we can set the rules to do what we want.